runtime/doc/autocmd.txt

```diff
@@ -722,6 +724,16 @@ InsertCharPre			When a character is typed in Insert mode,
 				It is not allowed to change the text |textlock|.
 				The event is not triggered when 'paste' is
 				set.
+							*TextYankPost*
+TextYankPost			Just after a |yank| or |deleting| command is
+				issued.  Not issued if the black hole register
+				(|quote_|) is used.  The pattern must be * as the
+				matched value might be changed in the future.
```

Comment:
don't need parens around the tag reference. Suggestion:

```
TextYankPost>   >       >       Just after a |yank| or |deleting| command is
>       >       >       >       issued.  Not issued if the black hole register
>       >       >       >       |quote_| is used.  Pattern must be * because
>       >       >       >       its meaning may change in the future.
>       >       >       >       It is not allowed to change the text |textlock|.
>       >       >       >       Sets these |v:event| keys:
>       >       >       >       >       operator
>       >       >       >       >       regcontents
>       >       >       >       >       regname
>       >       >       >       >       regtype
```

This format should be imitated by other autocmds which implement `v:event` in the future.